public/app/features/plugins/admin/components/PluginListRow.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { Icon, useStyles2, CardContainer, HorizontalGroup, VerticalGroup, Tooltip } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { CatalogPlugin, IconName } from '../types';
+import { PluginLogo } from './PluginLogo';
+import { PluginListBadges } from './PluginListBadges';
+
+const LOGO_SIZE = '48px';
+
+type PluginListRowProps = {
+  plugin: CatalogPlugin;
+  pathName: string;
+};
+
+export function PluginListRow({ plugin, pathName }: PluginListRowProps) {
```

Comment:
I'm arguing with myself over maintenance vs complexity with this component duplication. I originally thought that css-grid could help here but elements lose their flow when they're assigned to a grid cell so they would all overlap each other. Maybe we reduce some duplication between row and card components by creating smaller components for the shared pieces?

Reply:
Tried to address this in https://github.com/grafana/grafana/pull/39466/commits/04656db2aa3dfa8654eec0d0d6402e5329e949ad.

You will tell how it went. 😄

Reply:
I think it went very well! 😄